server/routes/users.js

```diff
@@ -84,24 +80,6 @@ async function logUser(req, res) {
   }
 }
 
-async function authenticateToken(req, res, next) {
-  const { cookie } = req.headers
-  const token = getToken(cookie)
-
-  if (!token) {
-    return res.sendStatus(401)
-  }
-
-  jwt.verify(token, process.env.ACCESS_TOKEN, (error, user) => {
-    if (error) {
-      return res.sendStatus(403)
-    }
-    req.user = user
-
-    next()
-  })
-}
-
 async function getUserData(req, res) {
   const user = await User.findOne({ _id: req.user })
 
```

Comment:
@AStashevskaya Also looks like you can avoid calling `await newUser.save()` after `const newUser = new User(user)` in `createUser`, because no fields changed between this commands.